concurrency-limits-servlet/src/main/java/com/netflix/concurrency/limits/servlet/ServletLimiterBuilder.java

```diff
@@ -72,7 +72,63 @@ public ServletLimiterBuilder partitionByParameter(String name) {
     public ServletLimiterBuilder partitionByPathInfo(Function<String, String> pathToGroup) {
         return partitionResolver(request -> Optional.ofNullable(request.getPathInfo()).map(pathToGroup).orElse(null));
     }
-    
+
+    /**
+     * Bypass the limit if the value of the provided header name matches the specified value.
+     * @param name The name of the header to check.
+     *             This should match exactly with the header name in the {@link HttpServletRequest } context.
+     * @param value The value to compare against.
+     *              If the value of the header in the context matches this value, the limit will be bypassed.
+     * @return Chainable builder
+     */
+    public ServletLimiterBuilder bypassLimitByHeader(String name, String value) {
+        return bypassLimitResolver((context) -> value.equals(context.getHeader(name)));
+    }
+
+    /**
+     * Bypass limit if the value of the provided attribute name matches the specified value.
+     * @param name The name of the attribute to check.
+     *             This should match exactly with the attribute name in the {@link HttpServletRequest } context.
+     * @param value The value to compare against.
+     *              If the value of the attribute in the context matches this value, the limit will be bypassed.
+     * @return Chainable builder
+     */
+    public ServletLimiterBuilder bypassLimitByAttribute(String name, String value) {
+        return bypassLimitResolver((context) -> value.equals(context.getAttribute(name).toString()));
+    }
+
+    /**
+     * Bypass limit if the value of the provided parameter name matches the specified value.
+     * @param name The name of the parameter to check.
+     *             This should match exactly with the parameter name in the {@link HttpServletRequest } context.
+     * @param value The value to compare against.
+     *              If the value of the parameter in the context matches this value, the limit will be bypassed.
+     * @return Chainable builder
+     */
+    public ServletLimiterBuilder bypassLimitByParameter(String name, String value) {
+        return bypassLimitResolver((context) -> value.equals(context.getParameter(name)));
+    }
+
+    /**
+     * Bypass limit if the request URI matches the specified URI.
+     * @param uri The URI to check against the {@link HttpServletRequest } URI.
+     *            If the request's URI matches this URI, the limit will be bypassed.
+     * @return Chainable builder
+     */
+    public ServletLimiterBuilder bypassLimitByRequestUri(String uri) {
+        return bypassLimitResolver((context) -> uri.equals(context.getRequestURI()));
+    }
```

Comment:
Nit: The full URI may be a little too specific to be useful 😀

Reply:
Agreed, replaced with path